# Question
Look at this function:

```js
const bailOut = () => {
  for (let i = 0; i < 5; i++) {
    if (i > 3) return;
    console.log('Inside the loop', i);
  }
  console.log('Outside the loop');
};
```

Is that `console.log('Outside the loop');` ever going to run? Why or why not?

# Answer
The console.log('outside the loop') will never run as long as it on the outside of the for loops function. 

# Teacher Feedback
console.log('outside the loop') will never run because the moment i > 3, it will hit the return statement to abandon the loop entirely
